src/api/vdom.js

```diff
@@ -89,7 +89,7 @@ const attributesContext = propContext(attributes, {
 
   // Default attribute applicator.
   [symbols.default] (elem, name, value) {
-    const { props, prototype } = customElements.get(elem.tagName) || {
+    const { props, prototype } = customElements.get(elem.localName) || {
```

Comment:
Fun fact, seems `localName` returns lowercase. We should be using it anyways, but it's a bonus it's lowercase.